lib/services/nativescript-project-plugins-service.ts

```diff
@@ -567,6 +611,85 @@ export class NativeScriptProjectPluginsService implements IPluginsService {
 		}).future<IBasicPluginInformation>()();
 	}
 
+	private setPluginInPackageJson(pluginIdentifier: string, pluginOpts?: {addPluginToPackageJson: boolean}): IFuture<void> {
+		return ((): void => {
+			let pathToPackageJson = this.getPathToProjectPackageJson().wait(),
+				packageJsonContent = this.getProjectPackageJsonContent().wait(),
+				pluginBasicInfo = this.getPluginBasicInformation(pluginIdentifier),
+				name = pluginBasicInfo.name,
+				selectedVersion = pluginBasicInfo.version || "latest",
+				basicPlugin = this.getBasicPluginInfoFromMarketplace(name, selectedVersion).wait() ||
+								this.getBasicPluginInfoFromNpm(name, selectedVersion).wait() ||
+								this.getBasicPluginInfoFromUrl(pluginIdentifier).wait();
+
+			if(!basicPlugin) {
+				this.$errors.failWithoutHelp(`Unable to add plugin ${pluginIdentifier}. Make sure you've provided a valid name, path to local directory or github URL.`);
```

Comment:
Maybe it should be `git` URL as we support urls like `https://github.com/Icenium/icenium-cli.git`
There's no requirement to be `github` only as far as I know. What do you think?